content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -43,75 +44,138 @@ Unsuitable content includes:
 
 To submit a blog post, follow these steps:
 
-1. [Sign the CLA](https://kubernetes.io/docs/contribute/start/#sign-the-cla) if you have not yet done so.
-1. Have a look at the Markdown format for existing blog posts in the [website repository](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts).
+1. [Sign the CLA](https://github.com/kubernetes/community/blob/master/CLA.md)
+   if you have not yet done so.
+
+1. Have a look at the Markdown format for existing blog posts in the
+   [website repository](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts).
```

Comment:
I think it would be better to use `git.k8s.io` for all `https://github.com/kubernetes/....` link.
```suggestion
   [website repository](https://git.k8s.io/website/content/en/blog/_posts).
```

Reply:
I don't understand why 'git.k8s.io' is better.